starknet_py/net/client.py

```diff
@@ -187,13 +187,16 @@ async def estimate_fee(
         tx: AccountTransaction,
         block_hash: Optional[Union[Hash, Tag]] = None,
         block_number: Optional[Union[int, Tag]] = None,
+        *,
+        skip_validate: bool = False,
```

Comment:
I'd first try to get confirmation from starkware that this will be included in the RPC at some point. If not, this argument should not be a part of the interface.